hack/tools/configure_vault_single_cluster.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+# Copyright 2020 IBM Corp.
+# SPDX-License-Identifier: Apache-2.0
+
+set -x
+
+op=$1
+
+source ./common.sh
+source ./vault_utils.sh
+
+# This script configures Vault server deployed in the control-plane in a single cluster setup.
+# It defines Vault role for modules running in m4d-blueprints namespace to authentication against
+# Vault to retrieve dataset credentials.
+# To do that The following is done:
+# - enable kubernetes auth path in path called kubernetes.
+# - enable Vault kv secret engine to hold the dataset credentials
+# - creation of Vault policy to allow to access dataset credentials in the Vault kv secret engine enabled above
+# - creation of Vault role for kubernetes auth path to bind policy to identity. The identity is the service accounts
+#  in m4d-blueprints namespace and the policy is the policy to allow them to access the
+#  path to the dataset credentials.
+
+: ${KUBE_NAMESPACE:=m4d-system}
+: ${MODULE_NAMESPACE:="m4d-blueprints"}
+: ${SECRET_PATH:=m4d/dataset-creds}
```

Comment:
Same comment as before. Where are Egeria credentials stored?

Reply:
The user inserts Egeria credentials to external path but then they are copied to SECRET_PATH